adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -810,6 +811,8 @@ private IWindowComponent wrapActivity(final Activity activity) {
         }
 
         return new IWindowComponent() {
+            //CHECKSTYLE:OFF: checkstyle:MemberName
+            //CHECKSTYLE:OFF: checkstyle:VisibilityModifier
             Activity mRefActivity = activity;
```

Comment:
@heidijinxujia it has nothing to do with test passed or not. What @addev-ashish means is that we can make the mRefActivity to be private then we can get rid of the comment to suppress checkstyle.